src/api/app.py

```diff
@@ -25,8 +29,16 @@
 courses = Courses.Courses(db_conn)
 date_range_map = DateMapping.semester_date_mapping(db_conn)
 admin_info = AdminInfo.Admin(db_conn)
+semester_info = SemesterInfo.semester_info(db_conn)
+users = UserModel.User()
 
 app = Flask(__name__)
+app.secret_key = "someRandomStuffForNow"
```

Comment:
Please make this an env var. We should not have exposed keys/secrets.
- [ ] Fetch with `os.environ`
- [ ] Add env var in dev & prod docker-compose with default like `${SECRET:-default_key}`, should be a few examples in docker-compose.production.yml

Reply:
Done